lib/advisor/web/controllers/progress_page.ex

```diff
@@ -1,16 +1,16 @@
 defmodule Advisor.Web.ProgressPage do
   use Advisor.Web, :controller
-  alias Advisor.Core.{People, AdvisoryFinder, AnswerFinder, QuestionnaireFinder}
+  alias Advisor.Core.{People, AdviceFinder, Answers, Questionnaire}
 
   plug  Advisor.Web.Authentication.Gatekeeper, only: :group_leads
 
   def index(conn, %{"id" => id}) do
-    advisories = AdvisoryFinder.gather_for_questionnaire(id)
-    questionnaire = QuestionnaireFinder.find(id)
-    requester = People.find_requester(questionnaire)
+    advisories = AdviceFinder.gather_for_questionnaire(id)
+    questionnaire = Questionnaire.find(id)
+    requester = People.requester(questionnaire)
```

Comment:
This is where keeping things very finely grained can cause an issue I think. You end up with more things on this level which is quite high.

You could potentially hide a lot of this behind something like `Progress.for_questionnaire(id)` for example.